client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -204,10 +196,6 @@ const NewsletterSetup: Step = ( { navigation } ) => {
 					enableAutoFocus={ false }
 					onChange={ onChange }
 					style={ {
-						backgroundImage: getBackgroundImage( tagline ),
-						backgroundRepeat: 'no-repeat',
-						backgroundPosition: '95%',
-						paddingRight: ' 40px',
 						paddingLeft: '14px',
```

Comment:
Do we need the padding?

Reply:
Yes, I have moved into css, makes more sense.